File: BeeCrowd/Cpp/beecrowd_1189.cpp
#include <stdio.h>

int main() {

  int i, j, count = 0;
  float x, add = 0;
  char tag;

  scanf("%c", &tag);

  for (i = 0; i < 12; ++i) {
    for (j = 0; j < 12; ++j) {
      scanf("%f", &x);
      if (i > j && j < 11 - i) {
        add += x;
        count += 1;
      }
    }
  }
  printf("%.1f\n", tag == 'S' ? add : add / count);

  return 0;
}

File: BeeCrowd/Python/beecrowd_1189.py
count, add = 0, 0
tag = input() 

for i in range(12):
    for j in range(12):
        x = float(input())
        if( i > j and j < 11 - i):
            add += x
            count += 1

print("%.1f" %(add if tag == 'S' else add/count) )
